public/extra_descriptions/simvi.html

```diff
@@ -0,0 +1,18 @@
+<link rel="stylesheet" href="../../vendor/css/bootstrap.min.css" />
+
+<p style="margin-top: 20px; font-weight: bold">
+  simvi - simple vi mode
+</p>
+
+<p>from tekezo's vi, dosen't work with mac os's combination move keys</p>
+<p>so I added keysets like below</p>
+<p>for work with mac os's combination keys</p>
+
+<ul>
+  <li>option + s + h,j,k,l</li>
+  <li>shift + option + s + h,j,k,l</li>
+  <li>command + s + h,j,k,l</li>
+  <li>shift + command + s + h,j,k,l</li>
```

Comment:
```suggestion
  <li>option + s + h,j,k,l</li>
  <li>shift + option + s + h,j,k,l</li>
  <li>command + s + h,j,k,l</li>
  <li>shift + command + s + h,j,k,l</li>
```

@rationality6 In case you have missed my question, I was asking about these. Shouldn't `s` be pressed first (before other keys)?

Reply:
@MuhammedZakir The ordering doesn't matter. both ways works fine!